x-pack/plugins/infra/common/log_sources/resolved_log_source_configuration.ts

```diff
@@ -119,7 +119,7 @@ const resolveRuntimeMappings = (indexPattern: DataView): estypes.MappingRuntimeF
           ? {
               script: {
                 lang: 'painless', // required in the es types
-                source: runtimeFieldSpec.script.source,
+                source: (runtimeFieldSpec.script as estypes.InlineScript).source,
```

Comment:
Since `runtimeFields` now corresponds to the mapping due to the changes in `data_view.ts`, no conversion is necessary anymore. This whole function should just be replaced with

```typescript
const resolveRuntimeMappings = (indexPattern: DataView): estypes.MappingRuntimeFields => {
  return indexPattern.getRuntimeMappings();
};
```

Thanks for introducing that shortcut!

Reply:
@weltenwort I implemented your suggestion...at least I think I did...but now some tests are failing and I'm quite confused as to what might be expected. Do you mind taking a look?

Reply:
Sorry for the delay. I took the liberty of pushing a fix for the tests. The assertion had to be adjusted slightly because we're not manually injecting the `"lang": "painless"` anymore, but instead we're passing through the mappings provided by the data view.